net-misc/pingu/pingu-1.5.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd tmpfiles toolchain-funcs
+
+DESCRIPTION="Policy routing daemon with failover and load-balancing"
+HOMEPAGE="https://github.com/ncopa/pingu"
+SRC_URI="https://github.com/ncopa/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc"
+
+RDEPEND="dev-libs/libev"
+DEPEND="${RDEPEND}
+	sys-kernel/linux-headers
+	virtual/pkgconfig
+	doc? ( app-text/asciidoc )"
+
+src_prepare() {
+	default
+
+	# Fix compilation issue
+	sed -i '/icp->un.frag.__unused = 0;/d' src/icmp.c \
+		|| die "sed failed for src/icmp.c"
+
+	# Fix QA with install into path /run/pingu must be created at runtime
+	sed  -i -e '/pingustatedir = /s|$(rundir)/pingu|/run/pingu|' \
+		-e '/$(DESTDIR)\/$(bindir) $(DESTDIR)\/$(sbindir)/s|.$||' \
```

Comment:
What is this doing exactly? It looks really suspicious.

Reply:
1. Changes pingustatedir to /run/pingu.
By default pingustatedir is /usr/var/run. I need it to be /run/pingu for pid.
2. Prohibits installation into /run dir (portage compliances about it).
It tries to install into /run, I removed "$(DESTDIR)/$(pingustatedir)" and previous line since it has "\" at the end.

Reply:
It's horribly unreadable, fragile and if this ever got a new release, it might just silently start doing horrible things. So don't. if you need to, use a patch instead.

Reply:
If patch, then patch.
Moved to patch instead sed.